src/components/ProgressBar.tsx

```diff
@@ -162,87 +128,90 @@ class ProgressBar extends React.Component<Props, State> {
       useNativeDriver: true,
       isInteraction: false,
     }).start();
-  };
+  }, [fade, scale]);
+
+  React.useEffect(() => {
+    if (visible) startAnimation();
+    else stopAnimation();
+  }, [progress, visible, startAnimation, stopAnimation]);
```

Comment:
We can remove progress (ESLint should hint you on that) as it's a dependency of `startAnimation`
```suggestion
  }, [visible, startAnimation, stopAnimation]);
```

Reply:
it's strange, but ESLint hinted only about missing dependencies in useEffect(). I'll check manually other components then

Reply:
Ah yea, I tend to remove dependencies and regenerate them once in a while. Sounds like something to be improved in the plugin :)